test/helpers/kubectl.go

```diff
@@ -4377,7 +4377,15 @@ func (kub *Kubectl) AddIPRoute(nodeName, subnet, gw string, replace bool) *CmdRe
 	}
 	cmd := fmt.Sprintf("ip route %s %s via %s", action, subnet, gw)
 
-	return kub.ExecInHostNetNS(context.TODO(), nodeName, cmd)
+	res := kub.ExecInHostNetNS(context.TODO(), nodeName, cmd)
+
+	if !replace && res.GetExitCode() != 0 &&
+		strings.Contains(res.GetStdErr().String(), "File exists") {
+
+		kub.ExecInHostNetNS(context.TODO(), nodeName, "ip route list")
+	}
```

Comment:
Couldn't we instead put this debug information in the `AfterFailed` block? If we add it here, then I think we need to file an issue to remember to remove it once the flake is solved.

Reply:
Yep, we could but I think it's better to execute `ip route list` immediately after we encounter the EEXIST error.

Reply:
Then you're looking for `JustAfterFailed` :slightly_smiling_face:

Reply:
Yeah, but I cannot just to add `(Just)AfterFailed` into the helper function, and I want to have `ip r l` for each failing invocation.